pkg/p2p/peer/peer.go

```diff
@@ -194,35 +194,30 @@ func (p *Reader) Accept() error {
 // Create two-way communication with a peer. This function will allow both
 // goroutines to run as long as no errors are encountered. Once the first error
 // comes through, the context is canceled, and both goroutines are cleaned up.
```

Comment:
This comment is no longer relevant. There is no visible error being thrown nor handled directly by `Create`